README.md

```diff
@@ -42,6 +38,17 @@ Documentation | Package
 
 Read the [contributing guidelines](CONTRIBUTING.md) file if you are interested in contributing.
 
+## Getting Started
+
+  1. Install [Node.js and npm](https://nodejs.org/en) <!-- TODO: use a dev container or NixOS maybe? -->
+      - Use the correct [engine version](package.json): [n](https://www.npmjs.com/package/n), [nvm](https://github.com/nvm-sh/nvm), or other
+  2. Use `pnpm` by default `corepack enable pnpm`
+  3. Install dependencies project `npm install`
```

Comment:
Typo here, should no `pnpm install`

Reply:
Will update. But I think it works either way after `corepack ...`.